prow/cmd/deck/static/pr-script.js

```diff
@@ -781,25 +786,10 @@ function createPRCard(pr, builds = [], queries = [], tidePools = []) {
 }
 
 /**
- * Load Github login button if user has not login.
+ * Redirect to initiate github login flow.
  */
-function loadGithubLogin() {
-    const button = document.createElement("BUTTON");
-    button.classList.add("mdl-button", "mdl-js-button", "mdl-button--raised",
-        "mdl-button--primary", "mdl-js-ripple-effect");
-    button.textContent = "Login to Github";
-    button.style.alignSelf = "center";
-    button.style.width = "160px";
-    button.addEventListener("click", () => {
-        const url = window.location;
-        window.location.href = url.origin + "/github-login";
-    });
-    const msg = createMessage(
-        "PR Status needs you to login and grant it OAuth scopes",
```

Comment:
explaining why we want you to login is useful, even if this specific message might need improvement.

having a button here also requires the user to acknowledge this instead of suddenly being redirected. It's not obvious that clicking "PR status" in the sidebar will lead to login immediately.

Reply:
The oauth app should be called "Prow PR Dashboard" instead of "PR Status", have an appropriate icon, and have a description explaining what it does. Gubernator does this and it works fine?

Reply:
I think just visiting the page and being redirected to a third party login form is still surprising behavior.

Separately we also need to handle when github login has *not* been configured for this deployment. I shouldn't need to configure a GitHub oauth app to deploy deck.

Reply:
If you don't have github oauth configured, the /pr endpoint is missing. We should fix the nav, too.

Reply:
Sure, but lazy loading a warning that into just the page is much more trivial than patching the nav. Users will either see the nav pop out or we'll have to rewrite deck to template the HTML (the latter is looking increasingly likely anyhow though).

Reply:
Removed from this PR, because it's irrelevant here :)